.github/workflows/docker-image.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Push to Docker Hub
         uses: docker/build-push-action@v1
```

Comment:
Can be updated to v3

Reply:
It's on the agenda. The upgrade introduces some breaking changes I don't want to  deal with right now